metaflow/plugins/datatools/s3/s3.py

```diff
@@ -682,7 +707,7 @@ def _list(keys):
 
         return list(starmap(S3Object, _list(keys)))
 
-    def info(self, key=None, return_missing=False):
```

Comment:
hmm, `.info()` with `key=None` should work, right?

Reply:
It has very little value but I'll switch it back.